src/sophia/Makefile

```diff
@@ -22,25 +22,15 @@ clean:
 distclean: clean
 	rm -f *.pyf
 
-.PHONY: remove_generated_files
-remove_generated_files:
-	rm -f *.pyf *.f2l *.so
-
-install: remove_generated_files
-$(OBJS): remove_generated_files
-
-$(LIB_DIR)/%$(LEXT): %$(LEXT)
-	$(COPY_COMMAND) $< $@	
-
-
 .PHONY: install
-install: $(LIB_DIR)/$(LIB_NAME)$(LEXT)
+install: $(LIB_NAME)$(LEXT)
+	$(COPY_COMMAND) *$(LEXT) $(LIB_DIR)
 
```

Comment:
Here the script always copies the library file even in the case when everything is up to date. Does it make sense to change it to the rule when copy is made only if the library file is regenerated?

Reply:
I don't know. There was some reason I made it this way. But I can't remember, it's many years ago. I'd rather stick to the current scheme because copies are quick. If we're gonna redo the build system anyways, then it will be obsolete soon. roght now it's just annoying that it rebuilds sophia every time.

Reply:
Then it is fine.